themes/bootstrap3/templates/RecordTab/holdingsils.phtml

```diff
@@ -120,17 +120,19 @@
               <? endif; ?>
             <? else: ?>
               <? /* Begin Unavailable Items (Recalls) */ ?>
-              <span class="text-danger"><?=$this->transEsc($row['status'])?><link property="availability" href="http://schema.org/OutOfStock" /></span>
+              <span class="text-danger"><?=$this->translate("HoldingStatus::" .$row['status'])?><link property="availability" href="http://schema.org/OutOfStock" /></span>
```

Comment:
I would advise against changing transEsc to translate here unless you intend for your translated output to include raw HTML code.

It may also merit discussion with others about whether it is appropriate to change the text domain here. What you are doing makes sense to me, but if people are currently relying on status translations in their main language file, switching this may cause confusion.